README.md

```diff
@@ -0,0 +1,51 @@
+# TempleApp
+
+This app is basically a cost-efficient way of maintaining records of poojas, donations made by people and also all the financial records in temples. Using this app, people in the temple counter will be able to enter all the details regarding the poojas they want or some donations they want to do for the temple. This app is connected to a printer so that it will print the details of pooja registrations of users.
+The app has a login interface and only some people like General secretary, president and treasurer of the temple can log in and be able to access the complete database.
+## Getting Started
+
+These instructions will get you a copy of the project up and be running on your local machine for development and testing purposes.
+### Prerequisites
+[Android Studio](https://developer.android.com/studio), with a recent version of the Android SDK.
+
+### Setting up your development environment
+
+- Download and install Git and add it to your PATH
```

Comment:
Do we need `add it to your PATH` ? That's sort of understood that is to be done when installing git.